scout/server/blueprints/variants/templates/variants/sv-variants.html

```diff
@@ -65,16 +65,17 @@
       <thead>
         <tr>
           <th style="width:8%">Rank</th>
```

Comment:
The sum of this is 93, why not 100?

Reply:
The table is fitting the screen so it doesn't have to get to 100, the percentage column ratios will be calculated anyway as if the total was 100. But you are right, it might be confusing to not have a percentage reaching 100, I changed it now!
Still looking ok:
<img width="1414" alt="Screen Shot 2020-09-04 at 08 31 57" src="https://user-images.githubusercontent.com/6919697/92207366-632c4f80-ee89-11ea-9460-7fdc43856db4.png">

Reply:
💯 !!